routes/api/communities/index.js

```diff
@@ -154,4 +154,19 @@ router.get('/:name/templates', (req, res) => {
     // TODO
 });
 
+/**
+ * Checks if community url is taken
+ */
+router.get('/:name/exists', async (req, res) => {
+    const name = req.params.name;
+
+    const exists = await models.Community.findOne({
+        where: models.sequelize.where(models.sequelize.fn('lower', models.sequelize.col('name')), name.toLowerCase())
+    });
+
+    if (!exists) {
+        return res.json({exists: !!exists});
```

Comment:
Currently the server only replies if it doesn't exist. This statement can be moved out of this if statement and return can also be omitted since it is the last line.

Reply:
Fixed!